docs/antora.yml

```diff
@@ -0,0 +1,23 @@
+name: asciidoctorj
+title: Asciidoctorj
```

Comment:
Please set the title of this component version to AsciidoctorJ (uppercase the J).

Reply:
I fixed also 2 more occurences of these.